tutorials/send-emails-tem-serverless-framework/index.mdx

```diff
@@ -0,0 +1,105 @@
+        npm install serverless -g
+        ```
+2. Clone the GitHub repository containing the configuration files needed to deploy your function:
+        ```
+        git clone !!!ADD REPO HERE!!
+        ```
+
+3. Access the repository and paste the following command to install the [Serverless plugin]():
+        ```
+        cd tem && serverless plugin install -n serverless-scaleway-functions
+        ```
+4. Copy and paste the following commands to export your environment variables. Make sure you add your **own variables**.
+        ```
+        export SCW_ACCESS_KEY=<Scaleway access key>
+        export SCW_SECRET_KEY=<Scaleway secret key>
+        export SCW_DEFAULT_ORGANIZATION_ID=<Scaleway default Organization>
+        export SCW_DEFAULT_PROJECT_ID=<Scaleway default Project>
+        export SCW_DEFAULT_REGION="fr-par"
+        export SCW_API_URL="https://api.scaleway.com"
+        ```
+
+## Add your variables
+
+1. In the `tem` repository, open the `serverless.yml` file and replace `$SCW_SECRET_KEY`, `$SCW_PROJECT_ID`, `$SCW_REGION` in the `provider` section with your own variables.
+2. Replace `$SCW_DEFAULT_ORGANIZATION_ID`, `$SCW_ACCESS_KEY` and `$SCW_SECRET_KEY` in the `secret` section with your own environment variables, then save your changes. Your `serverless.yml` file should look like the following:
+        ```
+        service: gomail
+        configValidationMode: off
+        singleSource: true
+        provider:
+            name: scaleway
+            runtime: go121
+            scwToken: f5dd7248-ebf1-4cea-abe6-60b58eb5df58
+            scwProject: f7d9b14a-c5fe-4e50-bb6e-4c7474ee611d
+            scwRegion: fr-par
+
+
+        plugins:
+          - serverless-scaleway-functions
+
+        package:
+            patterns:
+                - '!node_modules/**'
+                - '!.gitignore'
+                - '!.git/**'
+
+        functions:
+            gomail:
+              handler: Handler
+              memoryLimit: 1024
+              privacy: private
+              secret:
+                SCW_DEFAULT_ORGANIZATION_ID: f7d9b14a-c5fe-4e50-bb6e-4c7474ee611d
+                SCW_ACCESS_KEY: SCWDAC6A76BF6DC4EFD65
+                SCW_SECRET_KEY: f5dd7248-ebf1-4cea-abe6-60b58eb5df58
+
+        ```
+3. In the `main.go` file, replace `$YOUR_DOMAIN` with your domain that you have configured with Transactional Email, then save your changes.
+
+## Deploy your function
+
+1. In the same terminal as the previous steps, copy and paste the following command to trigger the deployment of your function:
+        ```
+        serverless deploy
+        ```
+2. Log in to your Scaleway account and click the Functions section in the left-side menu. Your `gomail` namespace displays.
+3. Click the `gomail` namespace, then click the `gomail` function.
+4. Click the **Endpoints** tab and copy your function's domain. Make sure you save it as you will need it in the next steps.
+5. Click the **Security** tab, then click **+ Generate token**.
```

Comment:
I think that you can link this help page: https://www.scaleway.com/en/docs/serverless/functions/how-to/secure-a-function/#restrict-access-to-your-functions